Jellyfin/Controls/JellyfinWebView.xaml.cs

```diff
@@ -31,16 +33,32 @@ public JellyfinWebView()
             _gamepadManager.OnBackPressed += HandleGamepadBackPress;
         }
 
-        private void HandleGamepadBackPress()
+        private async Task WView_NavigationStartingTask(WebView2 sender, CoreWebView2NavigationStartingEventArgs args)
         {
-            if (WView.CanGoBack)
+            // Workaround to fix focus issues with gamepad not focusing window
+            BtnFocusStealer.Visibility = Visibility.Visible;
+            BtnFocusStealer.Focus(FocusState.Programmatic);
+
+            // Force enabledGamepad on xbox only. 
+            if (AppUtils.IsXbox)
             {
-                WView.GoBack();
+                await WView.ExecuteScriptAsync("localStorage.setItem(\"enableGamepad\", \"true\")");
             }
         }
+        private async void WView_NavigationStarting(WebView2 sender, CoreWebView2NavigationStartingEventArgs args)
+        {
+            await WView_NavigationStartingTask(sender, args);
+        }
+
+        private void HandleGamepadBackPress()
```

Comment:
Might as well remove this event listener then if it is redundant.

Reply:
Good question. You can keep that file in the codebase. If it stays unused in the future, or gets superseeded by something else, we can always remove it when the time is right.